Aircraft.h

```diff
@@ -35,6 +36,7 @@ class Aircraft {
 
     void send(std::string);
     void send(DataLinkMessage );
+    void start_receiving();
```

Comment:
```suggestion
    void startReceiving();
```
can you please change this to camel case styling